src/helpers.coffee

```diff
@@ -31,13 +32,16 @@ jade = async ({source, target, data}) ->
   render = _jade.compile source.content, filename: source.path
   target.content = render data
 
+handlebars = async ({source, target, data}) ->
+  source.content ?= yield read source.path
+  target.content = _handlebars source.content, data
```

Comment:
I think maybe @dyoder's comment was referring to this line. Is panda-template already returning a promise?